### PR TITLE
Login query

### PR DESCRIPTION
**Pull request by GriffinWilchuk**
saving progress before switching to tasks suggested by TA

### CODE REVIEW

File: EquiFood-A-ReactNative/server/routes/customer.js [hunk 1]
```diff
@@ -33,6 +33,25 @@ router.post('/register',  async function(req,res){
     } catch (error) {
         res.status(400).send(error.message)
 }
-})
+});
 
-module.exports = router;
\ No newline at end of file
+uter.post('/login', async function(req,res){
```
Comment (MajinJake) [inline EquiFood-A-ReactNative/server/routes/customer.js:38]: Misspelled router lol

Comment (GriffinWilchuk) [reply EquiFood-A-ReactNative/server/routes/customer.js:38]: I don't know an uter post sounds pretty cool

Comment (GriffinWilchuk) [reply EquiFood-A-ReactNative/server/routes/customer.js:38]: thanks though

**Review by MajinJake — APPROVED** [review]

**Comment by MajinJake** [comment]
Should be good now

**Pull request merged**